source/server/server.cc

```diff
@@ -553,8 +556,6 @@ void InstanceImpl::initialize(const Options& options,
 
   // Runtime gets initialized before the main configuration since during main configuration
   // load things may grab a reference to the loader for later use.
-  runtime_singleton_ = std::make_unique<Runtime::ScopedLoaderSingleton>(
-      component_factory.createRuntime(*this, initial_config));
```

Comment:
Runtime instance creation depends on TLS thread registration. So it would be better to move initAccessLog after here than move runtime instance creation above.

Reply:
i got it